airflow/contrib/operators/adls_to_gcs.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.contrib.hooks.azure_data_lake_hook import AzureDataLakeHook
+from airflow.contrib.operators.adls_list_operator import AzureDataLakeStorageListOperator
+from airflow.utils.decorators import apply_defaults
+from airflow.contrib.hooks.gcs_hook import GoogleCloudStorageHook, _parse_gcs_url
+
+import os
+from tempfile import NamedTemporaryFile
+
+
+class AdlsToGoogleCloudStorageOperator(AzureDataLakeStorageListOperator):
+    """
+    Synchronizes an Azure Data Lake Storage path with a GCS bucket
+    :param path: The Azure Data Lake path to find the objects (templated)
```

Comment:
I see that you following a style of other similar operators like `S3ToGoogleCloudStorageOperator`. But what do you think about the name for this param: `src_adls` this would match with `dest_gcs`?

Reply:
Agreed, made this change